Jenkinsfile

```diff
@@ -94,7 +94,7 @@ pipeline {
 
     options {
         // preserve stashes so that jobs can be started at the test stage
-        preserveStashes(buildCount: 5)
+        preserveStashes(buildCount: 1)
```

Comment:
Why did you change this?

Reply:
I wasn't sure what this does. I was just curious if this has to do with the extra 4 'expected' builds that i removed from Jenkinsfile. So I just changed it to see what happens.